src/overmapbuffer.cpp

```diff
@@ -725,18 +680,22 @@ std::vector<tripoint> overmapbuffer::get_npc_path( const tripoint &src, const tr
     if( src == overmap::invalid_tripoint || dest == overmap::invalid_tripoint ) {
         return path;
     }
-    const tripoint start( OX, OY, src.z );   // Local source - center of the local area
-    const tripoint base( src - start );      // To convert local coordinates to global ones
-    const tripoint finish( dest - base );       // Local destination - relative to source
-    const auto get_ter_at = [&]( int x, int y ) {
-        x += base.x;
-        y += base.y;
-        const overmap &om = get_om_global( x, y );
-        return om.get_ter( x, y, src.z );
+
+
+    // Local source - center of the local area
+    const point start( OX, OY );
+    // To convert local coordinates to global ones
+    const tripoint base = src - start;
+    // Local destination - relative to base
+    const point finish = ( dest - base ).xy();
+
+    const auto get_ter_at = [&]( const point &p ) {
```

Comment:
```suggestion
    const auto get_ter_at = [&]( const point & p ) {
```

Reply:
Thanks.

<grumble>Astyle 3.1 was happy with it as it was.</grumble>